src/otx/recipe/detection/atss_mobilenetv2_tile.yaml

```diff
@@ -0,0 +1,186 @@
+# @package _global_
```

Comment:
recipe/detection/atss_mobilenetv2.yaml is updated. Could you check it?

Reply:
I updated the recipe. Thanks!